src/util/removeToolbarButton.ts

```diff
@@ -0,0 +1,41 @@
+import ShortcutId from '../@types/ShortcutId'
+import { alertActionCreator as alert } from '../actions/alert'
+import { deleteThoughtActionCreator as deleteThought } from '../actions/deleteThought'
+import { initUserToolbarActionCreator } from '../actions/initUserToolbar'
+import { AlertType, EM_TOKEN } from '../constants'
+import contextToPath from '../selectors/contextToPath'
+import findDescendant from '../selectors/findDescendant'
+import { getChildrenRanked } from '../selectors/getChildren'
+import { shortcutById } from '../shortcuts'
+import store from '../stores/app'
+
+/** Removes a toolbar button. */
+const removeToolbarButton = (shortcutId: ShortcutId) => {
```

Comment:
Despite the way this is defined in `CustomizeToolbar`, this should be an action-creator instead of a util function. It should be defined as a thunk so that we can dispatch it like so:

```ts
dispatch(removeToolbarButton(shortcutId))
```

I've found it's nice to make any changes to the Redux state explicit, and it allows them to be operated on by middleware.

While we're at it, let's have `CustomizeToolbar` import and use `removeToolbarButton` so that we don't have two versions of the function living in two different places.

Reply:
>this should be an action-creator instead of a util function.

Done: 709e2cf68b850366243fb8963688eb93c2b313a0

>While we're at it, let's have `CustomizeToolbar` import and use `removeToolbarButton` so that we don't have two versions of the function living in two different places.

Done: 3569733f965c989885b042ee93343d60a3142511